templates/single.twig

```diff
@@ -106,18 +106,12 @@
 					{% endfor %}
 				{% endif %}
 			</div>
+			{% include "blocks/author_profile.twig" with {post:post} %}
+
+			{% if ( show_comments ) %}
+				{% include "comments_section.twig" with {comments:post.get_comments()} %}
+			{% endif %}
 		</div>
 		<!-- Post Page Block End -->
-
-		{% include "blocks/author_profile.twig" with {post:post} %}
-
-		{% if ( post.articles ) %}
```

Comment:
We need to keep this functionality "Include articles in post" (option at the bottom of the post editor), so you should either keep everything here and put it in a specific div to give proper width, or move that conditional code with the rest